packages/wmr/README.md

```diff
@@ -7,177 +7,4 @@
-	}
-}
-```
-
-**5.** You're all set! As an extra step, if you'd like WMR to prerender your application to static HTML during production builds, replace `render()` with [preact-iso](https://www.npmjs.com/package/preact-iso):
-
-```diff
--import { render } from 'preact';
-+import { hydrate, prerender as ssr } from 'preact-iso';
-
-function App() {
-  return <h1>Hello World!</h1>;
-}
-
--render(<App />, document.body);
-+hydrate(<App />);
-
-+export async function prerender(data) {
-+  return ssr(<App {...data} />);
-+}
-```
-
-## Configuration and plugins
-
-WMR supports a `wmr.config.js` _(or `wmr.config.mjs`)_ configuration file, which can be used to set [WMR's options](https://github.com/preactjs/wmr/blob/main/packages/wmr/types.d.ts) and inject [Rollup plugins](https://github.com/rollup/plugins) or [Polka/Express middleware](https://github.com/lukeed/polka#middleware).
-
-You can export a `default` config function applied to all WMR commands, or individual functions for `start`, `build` and `serve`:
-
-```js
-// wmr.config.mjs
-import someRollupPlugin from '@rollup/plugin-xyz';
-
-/** @param {import('wmr').Options} config */
-export default async function (config) {
-	if (config.mode === 'build') {
-		config.plugins.push(
-			// add any Rollup plugins:
-			someRollupPlugin()
-		);
-	}
-
-	if (config.mode === 'serve') {
-		config.middleware.push(
-			// add any Polka middleware:
-			function myPolkaMiddleware(req, res, next) {
-				res.setHeader('X-Foo', 'bar');
-				next();
-			}
-		);
-	}
-}
-
-// Or configure each WMR command separately:
-export async function start(config) {
-	// equivalent to `config.mode === 'start'`
-}
-export async function build(config) {
-	// equivalent to `config.mode === 'build'`
-}
-export async function serve(config) {
-	// equivalent to `config.mode === 'serve'`
-}
-```
-
-> **Note:** remember to add `"type":"module"` to your package.json _or_ use the `.mjs` file extension to make the file a JS module.
-
-See [the full list of options](https://github.com/preactjs/wmr/blob/main/packages/wmr/types.d.ts).
-
-## Recipes
-
-Most applications can be built with WMR without any configuration or plugins.
-However, sometimes a bit of configuration can help optimize the output or simplify your development workflow.
-
-WMR supports Rollup plugins, and there's a growing list of [**configurations and recipes**](https://github.com/preactjs/wmr/wiki/Configuration-Recipes) in the wiki, including:
-
-- [minify the HTML output](https://github.com/preactjs/wmr/wiki/Configuration-Recipes#minifying-html)
-- [import directories of modules](https://github.com/preactjs/wmr/wiki/Configuration-Recipes#importing-directories-of-files)
-- [implement filesystem-based routing](https://github.com/preactjs/wmr/wiki/Configuration-Recipes#filesystem-based-routing--page-component-loading)
-- [add a service worker](https://github.com/preactjs/wmr/wiki/Configuration-Recipes#service-worker)
+[Read the Docs to learn more.](https://wmr.dev/docs)
```

Comment:
Might be good to leave the logo and quick bullets in this README since it's the npm package readme (I think?)

Reply:
Ohh, right haven't thought about that 👍